core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -687,7 +687,7 @@ trait EntityComponent {
     // Utility methods
 
     private def createAllAttributesString(entity: Entity): Option[String] = {
-      Option(s"${entity.name} ${entity.attributes.values.filterNot(_.isInstanceOf[AttributeList[_]]).map(AttributeStringifier(_)).mkString(" ")}".toLowerCase)
+      Option(s"${entity.name} ${entity.attributes.values.filterNot(_.isInstanceOf[AttributeList[_]]).map(AttributeStringifier(_)).mkString(" ")}".toLowerCase.take(65534))
```

Comment:
Why was this number chosen? Can we make it a constant and add an explanatory comment?

Reply:
it's the size of `TEXT` minus one because i'm scared. yes.